server/ts/leaderboard.ts

```diff
@@ -41,6 +41,35 @@ export const getLeaderboard = async (res: http.ServerResponse, body: string) =>
 	res.end(stringified);
 };
 
+/** Transmits all the scores for the marbleland mission specified by the id in the correct format as expected by Marbleland leaderboards integration */
+export const getMarblelandLeaderboard = async (res: http.ServerResponse, url: url.URL) => {
```

Comment:
I think I'd rename this to `getLeaderboardForMarbleland`, to make clear it's not Marbleland's leaderboard, we're just preparing it so that Marbleland can read it.